view/frontend/web/js/view/payment/method-renderer/storekeeper_payment.js

```diff
@@ -88,7 +88,7 @@ define(
                 return this.item.method == 'storekeeper_payment';
             },
             getPaymentIcon: function () {
-                var list = this.getPaymentMethodsList();
+                var list = window.checkoutConfig.storekeeper_payment_methods;
```

Comment:
is that correct? this.getPaymentMethodsList is not used anymore

Reply:
@ikulis This method called in knockout js data binding here: vendor/storekeeper/magento2-plugin/view/frontend/web/template/payment/storekeeper_payment.html:11

Reply:
Or wait, let me debug that